File: _Inventarios/RepMaxMinimos/DataSet1.vb
Partial Class DataSet1
    Partial Class DataTable1DataTable

        Private Sub DataTable1DataTable_ColumnChanging(ByVal sender As System.Object, ByVal e As System.Data.DataColumnChangeEventArgs) Handles Me.ColumnChanging
            If (e.Column.ColumnName = Me.OtrosEgresosColumn.ColumnName) Then

                'Agregar código de usuario aquí
            End If

        End Sub

    End Class

End Class

File: _Inventarios/RepCatalogo/DataSet1.cs
using System.Data;

namespace ArtInvent
{
	public partial class DataSet1
	{
		public partial class DataTable1DataTable
		{
			private void DataTable1DataTable_ColumnChanging(object sender, DataColumnChangeEventArgs e)
			{
				if ((e.Column.ColumnName ?? "") == (OtrosEgresosColumn.ColumnName ?? ""))
				{

					// Agregar código de usuario aquí
				}
			}
		}
	}
}